test/tools/runner/flaky.ts

```diff
@@ -0,0 +1,31 @@
+export const flakyTests = [
```

Comment:
One concern we discussed while refining NODE-6860 is that a hard-coded list of test names is brittle.  We discussed adding meta test or some sort of assertion to ensure that all the tests here are actual tests in our test suite.

Is that something you can add as well?